doc/howtos/pull-request-guidelines.rst

```diff
@@ -168,3 +168,7 @@ Dos and Don'ts
   happens only for the full set of commits that are getting merged, so
   when bisecting history, the merge commits are good candidates for
   testing because they are more likely to build correctly.
+
+* Never add backdoors (= unexpected access to the system), debug features,
```

Comment:
Do you have a convention for identifying and enabling these kinds of debug features?